docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md

```diff
@@ -0,0 +1,313 @@
+
+### 4) Set up behavior
+
+Enable the following behaviors by registering the plugins:
+
+| PLUGIN                                | SPECIFICATION                                                                                                                                                                                          | PREREQUISITES | NAMESPACE                                                          |
+|---------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|--------------------------------------------------------------------|
+| UnzerCheckoutDataResponseMapperPlugin | Maps `RestCheckoutDataTransfer.unzerCredentials.unzerKeypair.publicKey` to `RestCheckoutDataResponseAttributesTransfer.unzerPublicKey` while `RestCheckoutDataTransfer.unzerCredentials` is specified. | None          | SprykerEco\Glue\UnzerRestApi\Plugin\CheckoutRestApi                |
+| UnzerNotificationResource             | Adds Unzer notification resource.                                                                                                                                                                      | None          | SprykerEco\Glue\UnzerRestApi\Plugin\GlueJsonApiConventionExtension |
+
+**src/Pyz/Glue/CheckoutRestApi/CheckoutRestApiDependencyProvider.php**
+
+```php
+namespace Pyz\Glue\CheckoutRestApi;
+
+use Spryker\Glue\CheckoutRestApi\CheckoutRestApiDependencyProvider as SprykerCheckoutRestApiDependencyProvider;
+use SprykerEco\Glue\UnzerRestApi\Plugin\CheckoutRestApi\UnzerCheckoutDataResponseMapperPlugin;
+
+class CheckoutRestApiDependencyProvider extends SprykerCheckoutRestApiDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Glue\CheckoutRestApiExtension\Dependency\Plugin\CheckoutDataResponseMapperPluginInterface>
+     */
+    protected function getCheckoutDataResponseMapperPlugins(): array
+    {
+        return [
+            new UnzerCheckoutDataResponseMapperPlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Glue/GlueStorefrontApiApplication/GlueStorefrontApiApplicationDependencyProvider.php**
+
+```php
+namespace Pyz\Glue\GlueStorefrontApiApplication;
+
+use Spryker\Glue\GlueStorefrontApiApplication\GlueStorefrontApiApplicationDependencyProvider as SprykerGlueStorefrontApiApplicationDependencyProvider;
+use SprykerEco\Glue\UnzerRestApi\Plugin\GlueJsonApiConventionExtension\UnzerNotificationResource;
+
+class GlueStorefrontApiApplicationDependencyProvider extends SprykerGlueStorefrontApiApplicationDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Glue\GlueApplicationExtension\Dependency\Plugin\ResourceInterface>
+     */
+    protected function getResourcePlugins(): array
+    {
+        return [
+            new UnzerNotificationResource(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
```

Comment:
Add an introductory sentence explaining what we verify—for example, "Verify that the following API requests work:"